### ISSUE TITLE
Make "drop-items upon death" a configurable option on the server

### ISSUE THREAD

**Issue by Rushing** [Deceth/Battle-City#27]
Since Deceth has been able to convince me that longer lives = more fun, and that more playing = more fun, I came up with this idea, that's actually rather well thought out.
The backpack will be an item (One per city) that will be buildable after the Med-Kits.
Remember back when privates died, they used to keep all their items? Well this is much the same concept. People holding a backpack item will keep all items (Aside from the backpack) When they die.
Currently the server population is obviously down, and therefor there are alot of One vs Ones. These often result in stand still battles, and hardly go anywhere. The backpack item gives both attackers and defenders an advantage, with attackers being able to capatalize on a nice and effective attack, and defenders being able to get right back to defending after they are killed.

**Comment by Alton** [comment]
Hmm not bad ;P

**Comment by Deceth** [comment]
How many?

Or should this just be a server option (1 = lose items on death, 2 = keep items on death)

**Comment by Rushing** [comment]
I suggested the idea as being an item, but that's raised another interesting idea, the power to customize the number of each item as the server host.

**Comment by Deceth** [comment]
Any other name ideas for such an item?

**Comment by Rushing** [comment]
Backpack
Storage Compartment
Honestly, it's a pretty hard item to name. We could always go with the DFG style of naming, and call it the RTY =)

**Comment by Deceth** [comment]
Yes, let's call it the ORLY. Oh really, you thought I was dead huh!

**Comment by Rushing** [comment]
ORLY gets my vote =)

**Comment by Deceth** [comment]
Ok, so I've thought about this and I don't like the idea of this being an item. It's too confusing. People will pick it up, drop it, think about it, maybe shoot at it, but not know what it does. It doesn't add enough itself. Items should do something, without requiring other items to make them work.

That doesn't mean this is a bad idea. I think it should be open for discussion... when you blow-up... should you re-spawn with all your items, or not?

Right now, you re-spawn with nothing.

Back in the day, you really re-spawned with nothing. No laser or walkie. We got rid of those items and made them default so players can ALWAYS shoot and ALWAYS talk to their team. I think this was a great change, it solved a lot of problems, and was a huge help for new players.

... but now we're talking about the other items you pick-up. 

I prefer that you re-spawn without them. It gives other players a chance to grab the orb. It makes it a bad idea to grab all the items, because then you need to wait for the factory to re-make them. I think that's a fundamental part of the game.

That being said, I think it would be a good server option. Because if you host your own server, this is another way you can make yours unique. You could turn on this functionality.

So to clarify this issue:

**Make drop-items upon death a configurable option on the server** (this is not a priority... but it's the sort of thing we want to encourage going forward, as many server options as possible)
